doc/src/explanation/active-deprecations.md

```diff
@@ -76,7 +76,125 @@ SymPy deprecation warnings.
 
 ## Version 1.12
 
-There are no deprecations yet for 1.12.
+(deprecated-mechanics-joint-axis)=
+### New Joint intermediate frames
+
+The definition of the joint axis in the ``sympy.physics.mechanics`` has changed.
+Instead of using the arguments ``parent_axis`` and ``child_axis`` to
+automatically determine the joint axis and an intermediate reference frame. The
+joints now use an intermediate frame argument for both the parent and the child
+body, i.e. ``parent_interframe`` and ``child_interframe``. This means that you
+can now fully define the joint attachment, consisting of a point and frame, for
+both bodies. And if a joint like the ``PinJoint`` has a specific joint axis,
+e.g. the axis around which the rotation occurs, than this axis can be specified
+using the ``joint_axis`` argument. A big advantage of this setup is that you can
+be more accurately define the transformation from parent body to child body.
```

Comment:
```suggestion
using the ``joint_axis`` argument. An advantage of this setup is that one can
more accurately define the transformation from a parent body to a child body.
```

Reply:
would say "the parent body to the child body"